src/degradations/delay_tilde.c

```diff
@@ -102,11 +102,6 @@ void delay_adjust_buffer (t_delay_tilde * x) {
     return;
   }
 
-  if (delay_ms_inlet < 0) {
```

Comment:
That should not be removed.
The actual issue is the first cast in this function...
The incoming float might be < 0 and I was just casting it to unsigned int.
Casting should only be applied *after* checking for < 0....